system/gardener-customer-runtime/templates/seed.yaml

```
@@ -0,0 +1,18 @@
+{{- if .Values.seed }}
+---
+apiVersion: "openstack.stable.sap.cc/v1"
+kind: "OpenstackSeed"
+metadata:
+  name: gardener-customer-runtime-seed
+  labels:
+    app: "{{ .Release.Name }}"
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  requires:
+    - {{ .Release.Namespace }}/keystone-seed
+
+  domains:
+    - name: persephone
```

```suggestion
    - name: persephone
      description: ...
```

Do we need a dummy role assignment?